yapapi/golem.py

```diff
@@ -28,6 +28,37 @@
 
 
 class Golem(_Engine):
+    """
+    The main entrypoint of the Golem's high-level API.
+
+    Provides two methods that reflect the two modes of operation, or two types of jobs
+    that can currently be executed on Golem.
+
+    The first one - `execute_tasks` - instructs Golem to take a sequence of task fragments
+    that the user wishes to compute on Golem and distributes those among the providers.
+
+    The second one - `run_service` - instructs Golem to spawn a certain number of instances
```

Comment:
I feel like there's some ambiguity in the use of the word `Golem` here and in the two previous paragraphs. Is it `Golem` the Python class? Is it `Golem` as in local `yagna` node? Is it `Golem` as in _the network_?